### PR TITLE
chore(atomic): add tests + stories for atomic-commerce-product-list

### PR DESCRIPTION
**Pull request by fbeaudoincoveo**
https://coveord.atlassian.net/browse/KIT-3247

### CODE REVIEW

File: packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx [hunk 1]
```diff
@@ -4,28 +4,44 @@ import {
...
+      r.body = JSON.stringify(parsed);
+      return r;
+    },
+  },
 });
 
 const meta: Meta = {
+  argTypes: {
+    'attributes-display': {
+      options: ['grid', 'list', 'table'],
```
Comment (fbeaudoincoveo) [inline packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx:27]: I noticed that table display doesn't work at all. I'll create a Jira.

File: packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx [hunk 2]
```diff
@@ -4,28 +4,44 @@ import {
...
+      control: {type: 'radio'},
+    },
+    'attributes-density': {
+      options: ['compact', 'comfortable', 'normal'],
+      control: {type: 'radio'},
+    },
+    'attributes-image-size': {
+      options: ['small', 'large', 'icon', 'none'],
```
Comment (fbeaudoincoveo) [inline packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx:35]: At the moment, the atomic-commerce-product-children component doesn't honor imageSize: none. I'll create a Jira.

**Review by alexprudhomme — COMMENTED** [review]
Shouldn't we test a bunch more stuff like the combination of different layout props (display, density, imageSize) & the numberOfPlaceholders prop ?

File: packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx (hunk 2 above)
Comment (fbeaudoincoveo) [reply packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx:35]: See https://coveord.atlassian.net/browse/KIT-3608

File: packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx (hunk 1 above)
Comment (fbeaudoincoveo) [reply packages/atomic/src/components/commerce/atomic-commerce-product-list/atomic-commerce-product-list.new.stories.tsx:27]: See https://coveord.atlassian.net/browse/KIT-3640

**Pull request merged**